app/config/ConfigDecorator.scala

```diff
@@ -196,60 +201,64 @@ class ConfigDecorator @Inject()(environment: Environment, configuration: Configu
 
   // Links back to pertax
   lazy val pertaxFrontendHomeUrl = pertaxFrontendHost + routes.HomeController.index().url
-  lazy val pertaxFrontendBackLink = configuration
+  lazy val pertaxFrontendBackLink = runModeConfiguration
     .getString("external-url.pertax-frontend.host")
     .getOrElse("") + routes.HomeController.index().url
 
   // Links to sign out
   lazy val citizenAuthFrontendSignOut = citizenAuthHost + "/ida/signout"
 
   lazy val welshLangEnabled = langs.availables.exists(l => l.code == "cy")
-  lazy val taxCreditsEnabled = configuration.getString("feature.tax-credits.enabled").getOrElse("true").toBoolean
+  lazy val taxCreditsEnabled = runModeConfiguration.getString("feature.tax-credits.enabled").getOrElse("true").toBoolean
   lazy val activateSALinkEnabled =
-    configuration.getString("feature.activate-sa-link.enabled").getOrElse("true").toBoolean
+    runModeConfiguration.getString("feature.activate-sa-link.enabled").getOrElse("true").toBoolean
   lazy val allowLowConfidenceSAEnabled =
-    configuration.getString("feature.allow-low-confidence-sa.enabled").getOrElse("false").toBoolean
-  lazy val ltaEnabled = configuration.getString("feature.lta.enabled").getOrElse("true").toBoolean
-  lazy val urLinkUrl = configuration.getString("feature.ur-link.url")
-  lazy val platformFrontendHost = configuration.getString("platform.frontend.host").getOrElse("")
-
-  lazy val taxcalcEnabled = configuration.getString("feature.taxcalc.enabled").getOrElse("true").toBoolean
-  lazy val taxComponentsEnabled = configuration.getString("feature.tax-components.enabled").getOrElse("true").toBoolean
-  lazy val saReminderBannerEnabled = configuration.getString("feature.sa-banner.enabled").getOrElse("true").toBoolean
-  lazy val nispEnabled = configuration.getString("feature.nisp.enabled").getOrElse("true").toBoolean
-  lazy val allowSaPreview = configuration.getString("feature.allow-sa-preview.enabled").getOrElse("false").toBoolean
+    runModeConfiguration.getString("feature.allow-low-confidence-sa.enabled").getOrElse("false").toBoolean
+  lazy val ltaEnabled = runModeConfiguration.getString("feature.lta.enabled").getOrElse("true").toBoolean
+  lazy val urLinkUrl = runModeConfiguration.getString("feature.ur-link.url")
+  lazy val platformFrontendHost = runModeConfiguration.getString("platform.frontend.host").getOrElse("")
+
+  lazy val taxcalcEnabled = runModeConfiguration.getString("feature.taxcalc.enabled").getOrElse("true").toBoolean
+  lazy val taxComponentsEnabled =
+    runModeConfiguration.getString("feature.tax-components.enabled").getOrElse("true").toBoolean
+  lazy val saReminderBannerEnabled =
+    runModeConfiguration.getString("feature.sa-banner.enabled").getOrElse("true").toBoolean
+  lazy val nispEnabled = runModeConfiguration.getString("feature.nisp.enabled").getOrElse("true").toBoolean
+  lazy val allowSaPreview =
+    runModeConfiguration.getString("feature.allow-sa-preview.enabled").getOrElse("false").toBoolean
```

Comment:
Throughout this file we're using this getString.getOrElse.toBoolean pattern rather than getBoolean.getOrElse. Is there any particular reason we're doing this?